docs/screens/reviewers_screen.md

```diff
@@ -0,0 +1,41 @@
+| ![Success after notify](../analytics_imgs/reviewers/after_notify_success.png) | <pre>{<br />   "name": "action_finished"<br />   "item_name": "notify"<br />   "success": true<br />}<pre/> |
+
+|           After user clicks notify, action finished with error.           | Specification                                                                                                |
+|:-------------------------------------------------------------------------:|--------------------------------------------------------------------------------------------------------------|
+| ![Error after notify](../analytics_imgs/reviewers/after_notify_error.png) | <pre>{<br />   "name": "action_finished"<br />   "item_name": "notify"<br />   "success": false<br />}<pre/> |
+
+|                           When user pulls to refresh.                            | Specification                                                                                    |
+|:--------------------------------------------------------------------------------:|--------------------------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/reviewers/pull_to_refresh.png" width=50% height=50%> | <pre>{<br />   "name": "action_start"<br />   "item_name": "refresh_reviewers_data"<br />}<pre/> |
+
+|                       When refresh data finished with success.                        | Specification                                                                                                               |
+|:-------------------------------------------------------------------------------------:|-----------------------------------------------------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/reviewers/refresh_with_success.png" width=50% height=50%> | <pre>{<br />   "name": "action_finished"<br />   "item_name": "refresh_reviewers_data"<br />   "success": true<br />}<pre/> |
+
+|                       When refresh data finished with error.                        | Specification                                                                                                                |
+|:-----------------------------------------------------------------------------------:|------------------------------------------------------------------------------------------------------------------------------|
+| <img src="../analytics_imgs/reviewers/refresh_with_error.png" width=50% height=50%> | <pre>{<br />   "name": "action_finished"<br />   "item_name": "refresh_reviewers_data"<br />   "success": false<br />}<pre/> |
+
+|                         When user opens reviewers screen.                         | Specification                                                                                  |
```

Comment:
**chore**: `When user opens reviewers screen.` -> `When fetching data is started`